app/views/posts/show.html.erb

```diff
@@ -1,2 +1,24 @@
-<h1>Posts#show</h1>
-<p>Find me in app/views/posts/show.html.erb</p>
```

Comment:
Please make the layout of the post show page that displays post with all comments listed as shown on [this](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_single_post.png?token=GHSAT0AAAAAAB34BLWSDE3BKERP263UBM3QY4JGD6A) wireframe